src/modules/ie-objects/elasticsearch/queryBuilder.ts

```diff
@@ -243,7 +260,18 @@ export class QueryBuilder {
 			},
 		];
 
-		if (user.isKeyUser && !isNil(user.sector)) {
+		if (isConsultableRemote || isConsultableMedia) {
+			checkSchemaLicenses = [
+				...checkSchemaLicenses,
+				{
+					terms: {
+						schema_license: [IeObjectLicense.BEZOEKERTOOL_CONTENT],
+					},
+				},
+			];
+		}
+
+		if (user.isKeyUser && !isNil(user.sector) && isConsultableRemote) {
```

Comment:
in the if above you check:
```
if (isConsultableRemote || isConsultableMedia) {
```
but in this if, you only check one of the 2, is that intentional?
```
if (user.isKeyUser && !isNil(user.sector) && isConsultableRemote) {
```